src/components/AccessibilityRequestNextStep/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
@CMSgov/truss-easi Curious to field other folks thoughts.

IMO, this is such specific markup that it doesn't need its own component. I can't see this ever being re-used again, so I'd prefer to move it directly into where it's being used.

Reply:
I went ahead and removed the component and applied it directly inside the page view.